src/dlgConnectionProfiles.cpp

```diff
@@ -2068,6 +2069,8 @@ bool dlgConnectionProfiles::validateProfile()
             valid = false;
         }
 
+        validName = true;
```

Comment:
This change will make all names valid, even though our algorithm flagged the name as invalid before... Maybe we should initialize all valid* variables with true on start of this function, similar to the simple valid.

Reply:
Thanks, good point - fixed it up.